addons/addon-base/packages/services/lib/user/__tests__/user-authz-service.test.js

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+const ServicesContainer = require('@aws-ee/base-services-container/lib/services-container');
+const UserAuthzService = require('../user-authz-service');
+const SystemContext = require('../../helpers/system-context');
```

Comment:
Can you use const systemContext = , instead of const SystemContext= ?
Since SystemContext is not a class (right?), the standard conventions in javaScript are to use camel case for exported objects, and to use pascal case for classes.